__tests__/app.tests.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-env jest */
+const request = require('request')
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.port
+
+let browser
+let page
+
+const requestParamsHomepage = {
+  url: `http://localhost:${PORT}/`,
+  headers: {
+    'Content-Type': 'text/plain'
+  }
+}
+
+beforeAll(async (done) => {
+  browser = global.browser
+  page = await browser.newPage()
+  done()
+})
+
+afterAll(async (done) => {
+  await page.close()
+  done()
+})
+
+describe('Homepage', () => {
+  it('should resolve with a http status code of 200', done => {
+    request.get(requestParamsHomepage, (err, res) => {
+      expect(res.statusCode).toEqual(200)
+      done(err)
+    })
+  })
+
+  it('should resolve with a ‘Content-Type’ header of "text/html"', done => {
+    request.get(requestParamsHomepage, (err, res) => {
```

Comment:
Ditto above.

Reply:
Dropped the `app.tests.js` as it's not valuable for the static website.